### ISSUE TITLE
Please notify when pull request was updated

### ISSUE THREAD

**Issue by quietnumeric** [integrations/slack#1091]
When pull request (has been sent notification to Slack) was updated (editing title or body, adding new commit), plugin send nothing to Slack.
I hope plugin edit existing Slack message when pull request title or body were updated (like issues and reviews and comments were edited).
I hope plugin send new Slack message when new commit was added to pull request.
Colleague wants to know not only content of commit but addition of commit itself.

**Comment by elielr01** [comment]
I agree. It would be great to either update the same slack message (same as what it is currently done with the number of Checks passed) or just send a new PR message when a PR is updated.

Nothing gets updated after PR creation.
Example:
1. Create PR with failing Checks
2. Fixed PR and all Checks passed

Even if you fixed the PR and all Checks pass the second time, in Slack it will still show the outdated PR version.
